hypershift-operator/controllers/nodepool/metrics/metrics.go

```diff
@@ -31,10 +31,14 @@ const (
 	CountByHClusterMetricName = "hypershift_hostedcluster_nodepools" // What about renaming it to hypershift_cluster_nodepools ?
 	countByHClusterMetricHelp = "Number of NodePools for a given HostedCluster"
 
-	CoresCountByHClusterMetricName = "hypershift_cluster_cores"
-	CoresCountByHClusterMetricHelp = "Number of cores for a given HostedCluster. " +
+	VCpusCountByHClusterMetricName = "hypershift_cluster_vcpus"
+	VCpusCountByHClusterMetricHelp = "Number of virtual CPUs as reported by the platform for a given HostedCluster. " +
 		"-1 if this number cannot be computed." // Be careful when changing this metric as it is used for billing the customers
 
+	VCpusComputationErrorByHClusterMetricName = "hypershift_cluster_vcpus_computation_error"
+	VCpusComputationErrorByHClusterMetricHelp = "Defined if and only if hypershift_cluster_vcpus is cannot be computed and is set to -1. " +
```

Comment:
```suggestion
	VCpusComputationErrorByHClusterMetricHelp = "Defined if and only if " + VCpusCountByHClusterMetricName + " cannot be computed and is set to -1. " +
```

Reply:
Thanks for pointing that out; proposed change taken into account.